plugins/obs-webrtc/whip-service.cpp

```diff
@@ -1,7 +1,7 @@
 #include "whip-service.h"
 
-const char *audio_codecs[MAX_CODECS] = {"opus"};
-const char *video_codecs[MAX_CODECS] = {"h264", "hevc", "av1"};
+const char *audio_codecs[] = {"opus", nullptr};
+const char *video_codecs[] = {"h264", "hevc", "av1", nullptr};
```

Comment:
Since we're returning this across a C/C++ boundary, is `nullptr` valid here?

Reply:
https://github.com/obsproject/obs-studio/blame/dfd5798e6ca8dc72532e96239b8229f032d37d4f/UI/window-basic-main-dropfiles.cpp#L18 I just copied another example of "C stuff in C++ land". Nullptr will coerce correctly here, but happy to switch it to NULL. (I was just trying to keep nullptr's in cpp files).

Reply:
`nullptr` will be fine here. And once we go to C23 we'll have it there as well!